packages/zoltan/src/driver/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ TRIBITS_ADD_EXECUTABLE(
   NOEXEPREFIX
   SOURCES ${COMMON_DRIVER_HDRS} ${ZDRIVE_SOURCES}
   COMM serial mpi
+  CATEGORIES BASIC NIGHTLY PERFORMANCE
```

Comment:
@brian-kelley Thanks for retrofitting zoltan; I did not realize there were issues with its tests.
Why are there three categories here but only two in zoltan/test/CMakeLists.txt:150  below?    
Are categories also needed for the zCPPdrive executable below (line 95)?

Reply:
Good catch. After some reading about tribits I think I understand that the categories say how expensive tests are to run: basic <= continuous < nightly < heavy, and then performance is completely on its own (needs to be added explicitly, and excludes basic if you don't also ask for basic). Since "nightly" is redundant after making it "basic", I just deleted that (and added basic to the zoltan/test file).

The default category (basic) will be fine for zCPPdrive because it doesn't get used by any performance tests.